database/postgres/pg.js

```diff
@@ -22,41 +22,45 @@ pgModule.closePGDB = () => {
 
 pgModule.createPgAccount = async (username, password) => {
   if (!username || !password) return;
+  const usernameAlphabetical = username.replace("@", "").replace(".", "");
```

Comment:
Why is the `replace(".", "")` needed?

Reply:
maybe we should auto generate the username using a library that can compose an alphanumeric string for us.

Reply:
```
song@mail.zheng.club
song.mail@zheng.club
```

Build your own encoder? https://github.com/samswen/base62-util/blob/master/src/index.js#L4